### ISSUE TITLE
Results obtained using collocation method do not plot

### ISSUE THREAD

**Issue by SeanMatthewNolan** [Rapid-Design-of-Systems-Laboratory/beluga#62]
If a solution was found using collocation, the plotting functions included with Beluga do not work. The following error occurred:

```
Traceback (most recent call last):
  File "C:/Users/nolase/Documents/beluga_dev/beluga/examples/planarHypersonic/plot.py", line 46, in <module>
    plots.render()
  File "C:\Users\nolase\Documents\beluga_dev\beluga\beluga\visualization\BelugaPlot.py", line 78, in render
    plot.preprocess()
  File "C:\Users\nolase\Documents\beluga_dev\beluga\beluga\visualization\elements.py", line 89, in preprocess
    sol.prepare(problem_data, mesh_size=self.mesh_size, overwrite=True)
  File "C:\Users\nolase\Documents\beluga_dev\beluga\beluga\bvpsol\Solution.py", line 163, in prepare
    for arc_start, arc_end in arcs:
TypeError: 'NoneType' object is not iterable 
```

**Comment by msparapa** [comment]
I think this is a combination of two problems. The `arcs` refer to the path constrained trajectories, which don't work, and Collocation currently ignores `arcs` so it never sets the default value to `(0,)`. Whatever solution you come up with, the `Solution()` class needs to be better. Plan on having each "arc" being an individual `Trajectory()`, and `Solution()` stitches them together. Between path constraints and multiple shooting, I don't think there's any working feature that could test this at this point.

**Comment by msparapa** [comment]
Fixed in [ea550a9](https://github.com/Rapid-Design-of-Systems-Laboratory/beluga/commit/ea550a916457d0e5e7a427cf42e956f9d5e29878) (I wrote the wrong Ref # in the commit message). Turns out the issue was just how objects are constructed and various assumptions made about what object exist and don't exist.
